brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2054 Slider arrows toggle option

### Description
Following new options added replacing multi-button control for Arrow and Dots visibility 

- Arrows 
- Dots 

### Screenshots
![Edit Page “Slider demo” ‹ WordPress — WordPress 2022-11-17 at 9 43 45 AM](https://user-images.githubusercontent.com/10000232/202353716-e55283a9-bd11-4fa9-b418-dcc44525515a.jpg)


### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?

- Add a slider block to the page/post
- Go to Right side block settings panel 
- Navigate to General -> Slider section. 
- Arrows and Dots Toggle options will be provided. 
- Enable/Disable them to check the live preview. 
- Save changes and Visit the front end page to check if changes are reflected. 

### Checklist:
